source/MaterialXRuntime/Private/PrvPath.h

```diff
@@ -0,0 +1,61 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_PRVPATH_H
+#define MATERIALX_PRVPATH_H
+
+#include <MaterialXRuntime/Library.h>
+#include <MaterialXRuntime/RtObject.h>
+
+#include <MaterialXRuntime/Private/PrvElement.h>
+
+namespace MaterialX
+{
+
+class PrvPath
+{
+public:
+    PrvPath(PrvObjectHandle root, PrvObjectHandle obj);
+
+    bool isValid() const
```

Comment:
Is ti worthwhile to inline this and other functions below. Not sure it really matters much, but figured I'd check with you.

Reply:
From what I've read, inlining in this case should be handled for you, so perhaps this comment isn't important. Feel free to ignore.

Reply:
Yep, functions defined inside a class definition is implicitly marked inline.